### DISCUSSION TITLE
Why is yarn 1.x not on the benchmarks?

### DISCUSSION THREAD

**Discussion by ricardo-valero** [pnpm/pnpm#3523] [Q&A] [answered]
I wanted to know if pnpm could replace yarn (classic).
I get pnpm's motivation/philosophy, but is it faster?

**Comment by zkochan** [comment, marked as answer]
Yarn claims that v2 is faster. In the benchmarks, we have Yarn v2 with node-linker=node-modules on, which creates the node-modules.

Here are the benchmarks maintained by Yarn if you want to compare Yarn 1 and 2: https://yarnpkg.com/benchmarks

**Reply by ricardo-valero** [reply]
Maybe I'm out of the loop, but isn't yarn classic the current de facto choice? 
I just think having yarn v1 on the benchmarks would make changing to pnpm a better informed decision for users like me.
Well thanks for the other benchmarks.

**Reply by zkochan** [reply]
No, Yarn 1 is not actively maintained (maybe not maintained at all, I don't know). People have the wrong impression that Yarn 1 is superior to Yarn 2. Yarn 2 can be used the same way as Yarn 1 with some configuration.

**Reply by ricardo-valero** [reply]
Thanks for the clarification, I'm now using pnpm 😃
